api/README.md

```diff
@@ -0,0 +1,11 @@
+API text files
+==============
+
+Each file here lists the public API when built with some set of features
+enabled. To create these files run `../contrib/check-for-api-changes.sh`:
+
+Requires `cargo-public-api`, install with:
+
+    `cargo +stable install cargo-public-api --locked`.
```

Comment:
Also this can be a `bash` fenced block no?

Reply:
Done, thanks.